internal/auth0/client/resource.go

```diff
@@ -514,6 +515,98 @@ func NewResource() *schema.Resource {
 								},
 							},
 						},
+						"azure_blob": {
+							Type:        schema.TypeList,
+							Optional:    true,
+							Computed:    true,
+							MaxItems:    1,
+							Description: "Azure Blob Storage Addon configuration.",
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"account_name": {
+										Description: "Your Azure storage account name. Usually first segment in your " +
+											"Azure storage URL. e.g. `https://acme-org.blob.core.windows.net` would " +
+											"be the account name `acme-org`.",
+										Type:     schema.TypeString,
+										Optional: true,
+									},
+									"storage_access_key": {
+										Description: "Access key associated with this storage account.",
+										Type:        schema.TypeString,
+										Optional:    true,
+									},
+									"container_name": {
+										Description: "Container to request a token for. e.g. `my-container`.",
```

Comment:
```suggestion
										Description: "Container to request a token for, like `my-container`.",
```
The docs team's guidelines instruct not to use latin abbreviations.